tests/datasource/fluent/test_schemas.py

```diff
@@ -83,11 +83,17 @@ def _sort_lists(schema_as_dict: dict) -> None:
         """
         key: str
         value: Any
+
         for key, value in schema_as_dict.items():
             if key == "required":
                 schema_as_dict[key] = sorted(value)
             elif key == "anyOf":
-                schema_as_dict[key] = sorted(value, key=_sort_any_of)
+                if isinstance(value, list):
```

Comment:
@Kilo59 this was giving some errors were schema items were being passed as a list. Some additional checks were made to ensure they were a list of `dicts` and not `str` which can not be sorted. Could you have a look and make any additional appropriate changes?